src/styles/utilities/_mixins.scss

```diff
@@ -0,0 +1,19 @@
+@mixin main-text {
+  font-weight: 500;
+  font-size: 12px;
+}
+
+@mixin small-text {
+  font-weight: 400;
+  font-size: 12px;
+}
+
+@mixin extra-small-text {
+  font-weight: 400;
+  font-size: 10px;
+}
+
+@mixin large-text {
+  font-weight: 700;
+  font-size: 16px;
+}
```

Comment:
I think you could use only 1 mixin for all of this ways, just like this: `@mixin text($weight, $size){font-weight: $weight; font-size: $size;}` )

Reply:
As for me, separate mixins for different types of text are better than one main mixin)
With this approach it's easier to understand in code which one is used, and what text it is